src/patterns/atoms/typographic/quotes.hbs

```diff
@@ -0,0 +1,14 @@
+---
+name: Quote Elements
+description: |
+  A pull quote is an excerpt that has been pulled from a body of text used to highlight a key topic. A blockquote is a quotation that is set off from the main text typically used for long quotations.
+links:
+  "MDN <blockquote>": https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote
+order: 5
+---
+
+<blockquote class="mzp-c-quote">
```

Comment:
This shouldn't have a class since right now we're just styling the generic element. If we later decide to make a special component for pull quotes or some other special quote, that would get a class to distinguish it from plain `blockquote` elements.